app/templates/users/show.tpl

```diff
@@ -9,7 +9,11 @@
 
   <body>
 
-    <h1>Welcome, <username /></h1>
+    <h1>Welcome, <username />
+      <form style="display: inline-block;" action="/logout" method="post">
+        <input type="submit" value="Logout">
+      </form>
+    </h1>
```

Comment:
Only [phrasing content](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) (and not block-level content like forms) is allowed in heading tags -- so maybe put the logout button outside the header tag for now. It won't look perfect but it'll be semantically correct. :)